Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -401,26 +402,26 @@ DataProcessorSpec getCATrackerSpec(ca::Config const& config, std::vector<int> co
 
           const unsigned char* ptr = nullptr;
           int count = 0;
-          int lastFEE = -1;
+          rdh_utils::FEEIDType lastFEE = -1;
```

Comment:
`rdh_utils::FEEIDType` is `uint16_t`, initialisation with -1 should still be fine I guess

Reply:
Shouldn't be a problem, it just needs to be different feeid than anything else.